paper-input-container.html

```diff
@@ -515,8 +515,7 @@
     _handleValue: function(inputElement) {
       var value = this._inputElementValue;
 
-      // type="number" hack needed because this.value is empty until it's valid
```

Comment:
Ooof, you're right. I wonder if it's this change: https://github.com/PolymerElements/paper-input/commit/c0fd9403ec77b4f0db67ed3526482d4dfc9f06c3 :/